components/post_view/commented_on/commented_on.jsx

```diff
@@ -23,28 +23,32 @@ export default class CommentedOn extends PureComponent {
     }
 
     handleOnClick = () => {
-        const {actions, displayName} = this.props;
-
+        const {actions} = this.props;
+        const displayName = this.makeUsername();
         actions.updateSearchTerms(displayName);
         actions.showSearchResults();
     }
 
+    makeUsername = () => {
+        const postProps = this.props.post.props;
+        let username = this.props.displayName;
+        if (
```

Comment:
nit: I haven't seen this style in our code before. Is it new?

Reply:
I just made a function because I needed to use in two places in the component

Reply:
Sorry, my comment was ambiguous: I was referring to the indentation of dropping down the first if condition to a new line.

Reply:
It was existing that way I think as it exceeds 80 char limit someone might have made it to multiple lines. We don't have a eslint rule and this style exists in a few places. Curious to know your thoughts on this?

Reply:
0/5, mostly just haven't seen that style for a long time :)